cds_ils/migrator/ill/api.py

```diff
@@ -148,11 +162,14 @@ def clean_record_json(record):
         legacy_id=record.get("legacy_id"),
         library_pid=library_pid,
         patron_pid=get_patron_pid(record),
-        status=get_status(record),
+        status=status,
         type=get_type(record),
     )
 
     # Optional fields
+    if status == "CANCELLED":
+        new_record.update(cancel_reason="MIGRATED/UNKNOWN")
```

Comment:
Is it ok to use this value `"MIGRATED/UNKNOWN"`? Didn't we got read of it?

Reply:
we got rid of it only in the frontsite, for literature requests, not for orders and ILLs